src/components/setup-sharehouse/ScheduleSetting.tsx

```diff
@@ -0,0 +1,85 @@
+'use client';
+import { Button } from '@/components/ui/button';
+// import { LoadingSpinner } from '@/components/ui/loadingSpinner';
```

Comment:
Could you delete the commented-out lines?

Reply:
Thank you!